about.html

```diff
@@ -13,51 +13,80 @@
 
 <body>
     <div class="container about-page">
+
+        <header>
+            <a href="index.html" class="logo"><img src="img/lambda-black.png" alt="lambda_black_logo" /></a>
+
+            <nav>
+                <ul>
+                    <li>Home</li>
+                    <li><a href="about.html">About</a></li>
+                    <li>Products</li>
+                    <li>Blog</li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <section class="banner">
+            <img src="img/jumbo-about.png" alt="jumbo" />
+        </section>
      
-    About
+        <section class="top-content about">
```

Comment:
Instead of assigning this `about` class multiple times you could just have used the `about-page` class you have at the top of your body to style your about page.